docs/DevGuide/Travis.md

```diff
@@ -74,6 +74,26 @@ performed in the CHECK_FILES build.
 using a Linux OS, and the `AppleClang` compiler for `OS X`.
 * The `gcc Debug` build will fail if there are `doxygen` warnings.
 
+## How to perform the checks locally
+
+Before pushing to GitHub and waiting for Travis to perform the checks it is
+useful to perform at least the following tests locally:
+- **Unit tests:** Perform a `make test-executables` and then `ctest -L unit` to
+  run all unit tests. This can take a significant amount of time, so to run only
+  a subset of the tests you can use one of the other keywords that the tests are
+  labeled with, such as `ctest -L datastructures`. To run only a particular test
+  you can also execute `BUILD_DIR/bin/RunTests TEST_NAME` instead, where
+  `TEST_NAME` is the test identifier such as `Unit.DataStructures.Mesh`.
+- **clang-tidy:** Run `make clang-tidy FILE=SOURCE_FILE` where `SOURCE_FILE` is
+  a relative or absolute path to a `.cpp` file. Perform this check for the
+  source files that changed in this pull request.
+- **IWYU:** Also just for the changed files run `iwyu_tool.py -p . SOURCE_FILE
```

Comment:
You may want to add a warning that it is very easy to incorrectly install IWYU (if not using the Docker container) and generate nonsense errors